src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -504,9 +504,24 @@ def _deploy_arm_template_subscription_scope(cli_ctx,
                        deployment_name, properties, deployment_location)
 
 
-def _list_resources_odata_filter_builder(resource_group_name=None, resource_provider_namespace=None,
+def _list_resources_odata_filter_builder(cli=None, resource_group_name=None, resource_provider_namespace=None,
```

Comment:
Pls keep parameter consistent with original function. cli is not consistent.

Reply:
Done